Gemfile

```
@@ -1,5 +1,3 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
-gem 'jekyll-octicons', '>= 10.0.0', group: :jekyll_plugins
-
-gem 'faraday', '~> 0.17.3'
+gem 'jekyll-octicons', '~> 19.8.0', group: :jekyll_plugins
+gem 'jekyll', '~> 4.3.3'
```

Shall we also pin the Ruby version here while we're at it? 🤔 

In addition to the entry in `.ruby-version` (which would only be respected on systems running `rbenv`), we can store this explicitly in the `Gemfile` per https://bundler.io/v1.12/whats_new.html#ruby-version-locking:

```suggestion
gem 'jekyll', '~> 4.3.3'
ruby '~> 3.3.1'
```

---

Netlify uses `.ruby-version`. I use Docker and use `.ruby-version` to get the correct version. You use `rbenv`. No one else touches this repo. I think it's enough to have `.ruby-version.`